.vale/fixtures/RedHat/Spelling/testvalid.adoc

```diff
@@ -170,6 +177,7 @@ Mirantis
 Mixin
 Mixins
 Modularization
+Mongo DB
```

Comment:
It [looks like](https://www.google.com/search?q=Mongo+DB&sourceid=chrome&ie=UTF-8) "MongoDB" is the official spelling.

Reply:
Fixed.